shell/common/color_util.cc

```diff
@@ -6,44 +6,52 @@
 
 #include <vector>
 
-#include "base/strings/string_number_conversions.h"
 #include "base/strings/string_util.h"
 #include "base/strings/stringprintf.h"
+#include "content/public/common/color_parser.h"
+#include "ui/gfx/color_utils.h"
 
 namespace electron {
 
-SkColor ParseHexColor(const std::string& color_string) {
-  // Check the string for incorrect formatting.
-  if (color_string.empty() || color_string[0] != '#')
-    return SK_ColorWHITE;
+std::string SkColorToColorString(SkColor color, const std::string& format) {
+  double alpha_double = (double)(SkColorGetA(color)) / 255.0;
```

Comment:
_nit_ C-style cast is not needed

```suggestion
  const double alpha_double = double(SkColorGetA(color)) / 255.0;
```